src/test/resources/features/15-attachment/attachements.feature

```diff
@@ -0,0 +1,17 @@
+ Feature: Attachments Management
+
+   Background:
+     Given a user logs in, searches a patient and visits attachment page 
+     When a user clicks on attachment menu
+     Then the system loads the page
+
+   #User performing action on the attachment page
+   Scenario: Uploading attachment
+     When a user attaches a file
+     And a user enters a caption
+     And a user clicks "Upload file" functionality
```

Comment:
any reason for using double quotes in these statements

Reply:
Thanks @k-joseph  for the review. In the nutshell am addressing the comments and soon updating the PR